Make_Tables/mysqlconnect.py

```diff
@@ -1,6 +1,7 @@
 import mysql.connector
 import os
 from dotenv import load_dotenv
+from mysql.connector import Error as myerror
```

Comment:
- [ ] Rename instances of myerror as dberror/sqlerror or something similar (make the name more informative)

Reply:
Okay, I have renamed it sqlerror